utils/spdx/src/main/kotlin/SpdxExpression.kt

```diff
@@ -157,6 +157,12 @@ sealed class SpdxExpression {
      */
     protected open fun validChoicesForDnf(): Set<SpdxExpression> = setOf(this)
 
+    /**
+     * Return whether this expression contains present licenses, i.e. not all licenses in this expression are "not
```

Comment:
Use `"contains [present][SpdxConstants.isPresent] licenses"`, to help users that do not know what a "present" license is.